crm_lead_type/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
We need also sales man read permissions (select crm lead type in crm lead record). Have a look at sale_order_type ir.model.access.csv file:

https://github.com/OCA/sale-workflow/blob/13.0/sale_order_type/security/ir.model.access.csv

Reply:
![Bildschirmfoto_2020-12-30_13-22-10](https://user-images.githubusercontent.com/226753/103351005-186ebd00-4aa2-11eb-956b-fb9ac8147602.png)